test/features/wellknown_openidConfiguration.feature

```diff
@@ -0,0 +1,10 @@
+Feature: This endpoint is only used to facilitate OIDC provider details in a standardised manner.
+
+  Open ID Connect Dynamic Provider Discovery is not currently supported. 
+
+  Request endpoint: GET /.well-known/openid-configuration
+
+  Scenario: Successful facilitation of OIDC provider details
+    When The request to facilitate the OIDC provider details is sent
```

Comment:
Haha, I could not find Given in the js file, now I know why :D
I wonder if we want to leave it like that, because it's not a bug, but it's different from what we already wrote.
If we want to stick to our approach, please add Given but we can discuss it on Monday.